bench-routes.service

```diff
@@ -0,0 +1,12 @@
+#Service script for bench-route
+
+[Unit]
+Description= Systemd bench-routes service
+After=network-online.target network.target
+Requires=network-online.target network.target
+
+[Service]
+Type=simple
+RemainAfterExit=yes
+ExecStart=/home/ps/bench-routes 9090 start > /dev/null  2>&1
```

Comment:
I haven't followed the discussion yet, but what is this `ps` option here for? Is it a path for your own root directory?

Reply:
@aquibbaig  my bad. that's my username. my bench-routes binary is kept there.
'ExecStart' is the absolute path name of the executable that will run when the service is started.
It should be '/bin/bench-routes'.